### DISCUSSION TITLE
Bug of country code in Turkish localization

### DISCUSSION THREAD

**Discussion by kuyucuburak** [Daniel-Ioannou/flutter_country_picker#59] [General]
Hi, country code of US should be `US` not `ABD`. Country code should not be changed according to localizations. The others seems unchanged. It seems this is just a bug. Right?

https://github.com/Daniel-Ioannou/flutter_country_picker/blob/155adbab77bdadb24def42351136f8f0a5676949/lib/src/res/strings/tr.dart#L236

**Comment by Daniel-Ioannou** [comment]
Thanks yes is a bug. 
I also noticed that 'BT' code is duplicate in tr.dart
The second `BT` should be `IT` right?
https://github.com/Daniel-Ioannou/flutter_country_picker/blob/155adbab77bdadb24def42351136f8f0a5676949/lib/src/res/strings/tr.dart#L110

**Reply by kuyucuburak** [reply]
Yep, you are right.

**Comment by Daniel-Ioannou** [comment]
This has been fixed at v2.0.11.
